dbilo-siuntos-backend/Infrastructure/Models/Terminal.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Infrastructure.Models
+{
+    public class Terminal
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public Address Address { get; set; }
```

Comment:
Reusing address doesn't make sense (terminal doesn't have email, phone num, name, surname etc. + dependency cycle: Address has Terminal and Terminal has address.